### DISCUSSION TITLE
Docker port forced open fix

### DISCUSSION THREAD

**Discussion by marineboy0122** [teslamate-org/teslamate#2734] [Q&A]
Forcibly open ports 3000 and 4000 in docker.
Disabling docker closes the port.
I want to close the port to prevent external access

**Comment by cwanja** [comment]
A bit confused. You want the port closed indefinitely whether Docker is running or not? If that is the case, just make sure that there is no port forwarding on your router / modem. If the port is not open from the modem / router, there is no external access.
